packages/api/db/migration/20240202201923_update_animal_batch_permissions.js

```diff
@@ -0,0 +1,52 @@
+/*
+ *  Copyright (c) 2024 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+export const up = async function (knex) {
+  await knex.schema.alterTable('animal_batch_sex_detail', (t) => {
+    t.renameColumn('batch_id', 'animal_batch_id');
```

Comment:
Can we rename this to `animal_batch_id`? My intention is that we follow an X_id pattern where X is the name of the table the foreign key points to